index.html

```diff
@@ -1,10 +1,50 @@
 <!DOCTYPE html>
-<html>
+<html lang="en"> <!--added language just because-->
   <head>
     <meta charset="utf-8">
     <title>disa's portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     <link rel="stylesheet" href="vendor/styles/normalize.css"><!--to get rid of browser default settings-->
+    <link href="http://www.fontspace.com/aph-products/aphont" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles/base.css" />
+    <link rel="stylesheet" type="text/css" href="styles/layout.css" />
+    <link rel="stylesheet" type="text/css" href="styles/module.css" />
+    <link rel="stylesheet" type="text/css" href="styles/state.css" />
+    <link rel="stylesheet" type="text/css" href="styles/theme.css" />
+    <link rel="stylesheet" href="styles/icons.css"><!--I reeally want to use the icomoon icons, here. I'm still figuring out how-->
+    <link rel="icon" type="image/x-icon" href="images/umbrella-and-raindrops.png">
+
   </head>
   <body>
+    <section class="main">
+    <section class="header">
+        <h1>Say Something Cool</h1>
+        <h2>(with a custom font)</h2>
+        <nav class="main-nav">
+          <!-- contains page title and menu, which will be hamburger menu in its default state and nav bar in desktop view. I wonder if there's a cooler looking hamburger menu icon-->
+        <div class="icon-menu"></div>
+          <ul><!--list for menu. I really really want those icons. Figuring it out.-->
+            <li class="tab" data-content="projects"><a href="#" class="icon-home"> Home</a></li>
+            <li class="tab" data-content="about"><a href="#" class="icon-address-book"> About</a></li>
+            <li><a href="http://twitter.com/justdisa" target="_blank" class="icon-twitter3"> Twitter</a></li>
+            <li><a href="http://github.com/justdisa" target="_blank" class="icon-github"> GitHub</a></li>
+          </ul>
+        </nav>
+    </section>
+
+    <div class="project">
```

Comment:
You probably want to make this another `section`, rather than a `div`. This will make it easier to target all the sections later on (which you'll need to do since this will be a single-page app.